lib/paperclip/has_attached_file.rb

```diff
@@ -93,6 +94,12 @@ def add_active_record_callbacks
       @klass.send(:after_commit, :on => :destroy) { send(name).send(:flush_deletes) }
     end
 
+    def add_reflections
+      @klass._reflections.merge!(
+        { @name.to_s => ActiveRecord::Reflection::HasOneReflection.new(:photo, nil, {class: @klass.name}, @klass) }
```

Comment:
Line is too long. [115/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.